### ISSUE TITLE
feat(a11y-hcm): Sidebar Elements

### ISSUE THREAD

**Issue by Rudo-hayate** [Rudo-hayate/open-webui-a11y#24]
### Check Existing Issues

- [x] I have searched for all existing **open AND closed** issues and discussions for similar requests. I have found none that is comparable to my request.

### Verify Feature Scope

- [x] I have read through and understood the scope definition for feature requests in the Issues section. I believe my feature request meets the definition and belongs in the Issues section instead of the Discussions.

### Problem Description

1. Icon of the user (white font on orange background)

<img width="1906" height="933" alt="Image" src="https://github.com/user-attachments/assets/9a930bf4-ff3f-4e36-98f1-c8d87f9ff44f" />

<img width="1906" height="933" alt="Image" src="https://github.com/user-attachments/assets/19de2a16-ea27-4099-af46-65292fc8678b" />

2. On/Off switch with status OFF

<img width="1906" height="934" alt="Image" src="https://github.com/user-attachments/assets/7d03145b-8dbe-4b7f-b9ce-54acec04f1c0" />

### Desired Solution you'd like

Fix the contrast issues with better color contrasts - should be at minimum 3:1

### Additional Context

https://bitvtest.de/pruefschritt/bitv-20-web/bitv-20-web-9-1-4-3-kontraste-von-texten-ausreichend

**Comment by cguhl-adesso** [comment]
Additionally following points of the reports:
- increase contrast of badge colors (admin-panel)
- increase contrast of changelog
